packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -307,13 +296,6 @@ export default {
 			this.store.commit('toggleAllSelection')
 		},
 	},
-	provide() {
```

Comment:
This seems like it could potentially break the table. Why was this done and why is it safe?

Reply:
same as above